osu.Game/Screens/Edit/Screens/Compose/Layers/MaskContainer.cs

```diff
@@ -53,6 +57,9 @@ public override void Add(HitObjectMask drawable)
 
         public override bool Remove(HitObjectMask drawable)
         {
+            // Rider 2018.1 requires this (roslyn analyser issue?)
+            Debug.Assert(drawable != null);
```

Comment:
Have you reported this to them? Seems like a pretty weird scenario...

Reply:
Plan on doing so. No combination of `[NotNull]` helps, either. Could be roslyn, not r#.